client/public/index.html

```diff
@@ -7,6 +7,18 @@
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Seismic Risc" />
     <link rel="apple-touch-icon" href="logo192.png" />
+    <!-- Social Media -->
+    <meta name="description" content="Seismic Risc conține harta clădirilor cu grad ridicat de risc și resurse utile pentru consolidare." />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://seismicrisc" />
+    <meta property="og:title" content="Seismic Risc" />
+    <meta property="og:description" content="Seismic Risc conține harta clădirilor cu grad ridicat de risc și resurse utile pentru consolidare." />
+    <meta property="og:site_name" content="Seismic Risc" />
+    <meta property="og:image" content="https://seismic-risc.now.sh/share.png"> />
+    <meta property="og:image:url" content="https://seismic-risc.now.sh/share.png" />
```

Comment:
It would be better if you used `%PUBLIC_URL%` instead of adding the site address.